index.html

```diff
@@ -10,23 +10,48 @@
   </head>
   <body>
     <main>
-      <section>
-        <h1>All awesome books</h1>
-        <div class="container"></div>
-      </section>
-      <div class="bar"></div>
-      <section>
-        <h2>Add a new book</h2>
-        <form action="#">
-          <input class="title" type="text" placeholder="Title" required />
-          <br />
-          <br />
-          <input class="author" type="text" placeholder="Author" required />
-          <br />
-          <br />
-          <button class="add" type="submit">Add</button>
-        </form>
-      </section>
+      <div class="mainWrapper">
+        <header>
+          <a href="/" id="logo">Awesome Books</a>
+          <ul>
+            <li><a href="/">List</a></li>
+            <li><a href="/">Add new</a></li>
+            <li><a href="/">Contact</a></li>
+          </ul>
+        </header>
```

Comment:
- [ ] Please implement the time and date under the navigation as shown in the requirements wireframe 👍 
### Wireframe
![image](https://user-images.githubusercontent.com/46839548/183054619-c532918c-cf23-4ab3-86dd-850ba443f67d.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/183054817-e2d53bce-f344-4b4f-8bf6-7385b303fe63.png)